server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/GetMailboxes.feature

```diff
@@ -242,3 +242,17 @@ Feature: GetMailboxes method
     Then the mailbox "shared2" has 0 messages
     And the mailbox "shared2" has 0 unseen messages
 
+  Scenario: A sharee should be able to access a shared mailbox after it has been renamed by the owner
+    Given "alice@domain.tld" renames it's mailbox "shared" to "mySharedMailbox"
+    When "bob@domain.tld" ask for mailboxes
+    Then the mailbox "mySharedMailbox" should be present
+
+  Scenario: A sharer should be able to access its shared mailbox after it has been renamed by a sharee
+    Given "bob@domain.tld" renames the mailbox, shared by "alice@domain.tld", "shared" to "mySharedMailbox"
```

Comment:
I thought renaming a mailbox of an other user should not be allowed (for now)?

Reply:
I've checked by debugging, it's working... Free feature ;)

Reply:
it's not working it's just probably meaning that anyone can rename the folder of anyone. can you check please?